### ISSUE TITLE
[FEATURE REQUEST] Provide a calculated total consumption for Home 

### ISSUE THREAD

**Issue by Yorkie71** [flixlix/power-flow-card-plus#391]
**Is your feature request related to a problem? Please describe.**
This excellent add on fills the gaps of the built in energy card and the instant data is really useful. However, the built in card provides a cumulative total which is useful. I have added this to the solar circle using the `sensor.givtcp_saXXXXXX_pv_energy_today_kwh` entity and was going to do the same for home but apparently there is not a GivTCP entity that provides that data. In the built in card, I'm told that this is a calculated total.

**Describe the solution you'd like**
A way providing a cumulative total for the day that can be added to Home as a secondary entity would be very useful.
![image](https://github.com/flixlix/power-flow-card-plus/assets/9529010/fc0cc939-c921-4d90-8179-7537dae01137)

**Describe alternatives you've considered**
There are a number of GicTCP entities that look like they would be suitable but none of them provide the correct data.

**Comment by flixlix** [comment]
If by cumulative consumption you mean energy, then this card does not support it and there are currently no plans on supporting it (you can use my energy flow card plus for energy).
You can try doing this with the secondary info in home and changing the unit, but I don't guarantee it will work.

If by cumulative you mean solar+grid+battery, then that is already what is being done in the card.

**Comment by Yorkie71** [comment]
Hi @flixlix I managed to work out a way to display what I needed. I created a Reimann Sum integration taking data from the home consumption on my inverter and then created a meter helper which resets every day. Then I added this as a secondary item on the home element. Works great.
![Screenshot_20231028-164223](https://github.com/flixlix/power-flow-card-plus/assets/9529010/6f5c4457-e168-4f16-8dad-55f7654bee94)
